### DISCUSSION TITLE
input text box is unresponcive after update a value manually in livewire 3.0 in laravel 10

### DISCUSSION THREAD

**Discussion by techinfocomp** [livewire/livewire#6644] [7. Bugs (archived)]
### Livewire version

3.0

### Which PHP version are you using?

PHP 8.2

### Additional information

Laravel version 10.21.0
Livewire version 3.0
Using with Laravel Sail

### Steps To Reproduce

If you type the value manually in the input text box, then Livewire is unable to update the value dynamical.
We are using Livewire 3.0 for this example.

NOTE: One more issue when you remove `wire:model.live="count"` then both H1 and Input text increase and decrease value. 
_Eg: Suppose H1 value is 3, and we manually update input box value with 4 then Input box will not get update when we click on + or – button_

Livewire Component
```
<?php

namespace App\Livewire;

use Livewire\Component;

class Counter extends Component
{
    public $count = 1;

    public function increment()
    {
        $this->count++;
    }

    public function decrement()
    {
        $this->count--;
    }

    public function render()
    {
        return view('livewire.counter');
    }
}

```

Livewire View
```
<div>
    <h1>{{ $count }}</h1>

    <input type="text" wire:model.live="count" name="count" value="{{ $count }}" id="count">

    <button wire:click="increment">+</button>

    <button wire:click="decrement">-</button>
</div>
```

If we manually update the value of input text box then the counter stop for the input text box but still works for H1 tag.
But before that update input text box numeric value manually, every thing is working fine for both H1 tag and input text box.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment]
You need to remove `value="{{ $count }}"` as the value is already set by `wire:model.live`.
```
    <input type="text" wire:model.live="count" name="count" value="{{ $count }}" id="count">
```
